ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/BigIntegerModularExponentiationPrecompiledContract.java

```diff
@@ -31,14 +31,14 @@
 
   private static final BigInteger WORD_SIZE = BigInteger.valueOf(32);
   private static final BigInteger BITS_IN_BYTE = BigInteger.valueOf(8);
-  private static final BigInteger BASE_OFFSET = BigInteger.valueOf(96);
-  private static final BigInteger MAX_FIRST_EXPONENT_BYTES = BigInteger.valueOf(32);
-  private static final BigInteger GQUADDIVISOR = BigInteger.valueOf(20);
+  public static final BigInteger BASE_OFFSET = BigInteger.valueOf(96);
+  public static final BigInteger MAX_FIRST_EXPONENT_BYTES = BigInteger.valueOf(32);
```

Comment:
Should we move `MAX_FIRST_EXPONENT_BYTES`, `GQUADDIVISOR` and `MAX_GAS_BITS` to the Byzantium gas calculator?  Looks like they are used only in gas calculation.  `BASE_OFFSET` needs to stay, however.